types/underscore/index.d.ts

```diff
@@ -859,12 +865,22 @@ declare module _ {
          * The opposite of zip. Given a list of lists, returns a series of new
          * arrays, the first of which contains all of the first elements in the
          * input lists, the second of which contains all of the second
-         * elements, and so on.
+         * elements, and so on. (alias: transpose)
          * @param lists The lists to unzip.
          * @returns The unzipped version of `lists`.
          **/
         unzip(lists: List<List<any>>): any[][];
 
+        /**
+         * The opposite of zip. Given a list of lists, returns a series of new
+         * arrays, the first of which contains all of the first elements in the
+         * input lists, the second of which contains all of the second
+         * elements, and so on. (alias: unzip)
+         * @param lists The lists to transpose.
+         * @returns The transposed version of `lists`.
+         **/
```

Comment:
I'd be perfectly happy if you omitted the duplicate comment and just put `transpose` directly under `unzip`.

Reply:
Good idea!